### PR TITLE
gcc: add German translation

### PR DESCRIPTION
**Pull request by Abrynos**

### CODE REVIEW

File: pages.de/common/gcc.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`gcc {{Quelldatei1.c}} {{Quelldatei2.c}} -o {{Ausgabebinärdatei}}`
+
+- Erlaubt Warnungen und debug-Symbole in der Ausgabedatei:
+
+`gcc {{source.c}} -Wall -Og -o {{Ausgabebinärdatei}}`
+
+- Inkludiert Bibliotheken aus anderen Ordnern:
+
+`gcc {{Quelldatei.c}} -o {{Ausgabebinärdatei}} -I{{Pfad_zu_Headerdatei}} -L{{Pfad_zu_Bibliothek1}} -l{{Pfad_zu_Bibliothek2}}`
```
Comment (navarroaxel) [inline pages.de/common/gcc.md:16]: We use `/` for paths:
```suggestion
`gcc {{Quelldatei.c}} -o {{Ausgabebinärdatei}} -I{{Pfad/zu/Headerdatei}} -L{{Pfad/zu/Bibliothek1}} -l{{Pfad/zu/Bibliothek2}}`
```

File: pages.de/common/gcc.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# gcc
+
+> Präprozessiert und kompiliert C und C++ Quellcodedateien und linkt diese anschließend zusammen.
+> Mehr Informationen: <https://gcc.gnu.org>.
+
+- Kompiliert mehrere Quellcodedateien zu einer ausführbaren Datei:
+
+`gcc {{Quelldatei1.c}} {{Quelldatei2.c}} -o {{Ausgabebinärdatei}}`
```
Comment (navarroaxel) [inline pages.de/common/gcc.md:8]: We use `snake_case` for variable names, `Ausgabebinärdatei` requires start with mayus? the same for `Pfad`, etc.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @Abrynos, I left some comments.

File: pages.de/common/gcc.md (hunk 1 above)
Comment (Abrynos) [reply pages.de/common/gcc.md:16]: there was no slashes in the english original

File: pages.de/common/gcc.md (hunk 2 above)
Comment (Abrynos) [reply pages.de/common/gcc.md:8]: nouns in german start with uppercase

File: pages.de/common/gcc.md (hunk 1 above)
Comment (Abrynos) [reply pages.de/common/gcc.md:16]: https://github.com/tldr-pages/tldr/blob/master/pages/common/gcc.md

Comment (navarroaxel) [reply pages.de/common/gcc.md:16]: @Abrynos Yeah, I know, but I like your change. and I believe we should fix the English page too. Maybe you can add the edition in this PR if you want.

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Don't speak the language, but the syntax looks fine to me!

Thanks :slightly_smiling_face:

**Comment by sbrl** [comment]
I think @CLAAssistant is having a hiccup. I'll merge this by force.

**Pull request merged**